students/hyska_monika/lesson_11_argparse_math/regex_search.py

```diff
@@ -0,0 +1,60 @@
+"""
+Program that opens all .txt files in a folder and
+searches for any line that matches a user-supplied regular expression.
+The results should be printed to the screen.
+"""
+
+import re
+import os
+import argparse
+import sys
+
+
+def check_arg(args=None):
+    parser = argparse.ArgumentParser(description='Search regex on files')
+    parser.add_argument('-p', '--path',
+                        help='path to search',
+                        default='./')
+    parser.add_argument('-r', '--regex',
+                        required='True',
+                        help='regex to search')
+    results = parser.parse_args(args)
+    return (results.path,
```

Comment:
I recommend other identation style, that makes the work easier:
```
assignment = call(
   arg_after_4_spaces=blah_blah,
    ...
)
```
Some IDEs or editors give you hard time when you left-justify.